feature-libs/asm/core/facade/asm-bind-cart.service.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { AsmBindCartFacade, BindCartParams } from '@spartacus/asm/root';
+import { Command, CommandService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { AsmConnector } from '../connectors';
+
+@Injectable()
+export class AsmBindCartService implements AsmBindCartFacade {
```

Comment:
This class needs a corresponding unit test.  (even though the methods don't contain alot of logic)